cl.conf

```diff
@@ -161,15 +161,15 @@ storage:
             sleep 2
           done
 
-
-          export K8S_VERSION=1.9.6
+          export K8S_CTL_VERSION=v1.9.6
+          export K8S_VERSION=v1.9.10
           export MK=/usr/local/bin/minikube
           export KUBECTL=/usr/local/bin/kubectl
           USER_HOME=$(mount | grep nfs | grep -v 'var/amobee\|etc/amobee\|var/log/amobee' | awk '{ print $3 }')
           VAGRANT=${USER_HOME}/.bricks/clusters/vagrant
           
-          sudo curl -Lo $MK https://storage.googleapis.com/minikube/releases/v0.24.1/minikube-linux-amd64 && sudo chmod +x $MK
-          sudo curl -Lo $KUBECTL https://storage.googleapis.com/kubernetes-release/release/v${K8S_VERSION}/bin/linux/amd64/kubectl && sudo chmod +x $KUBECTL
+          sudo curl -Lo $MK https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && sudo chmod +x $MK
```

Comment:
The reason we never downloaded 'latest' was because of changes to minikube that began selecting the 10.0.2.15 interface as the default gateway. The result was that minikube would generate certificates for the apiserver and the generated kubeconfig that used '10.0.2.15' as the CN for the certificate rather than '127.0.0.1', which is what it used before. The '127.0.0.1' interface allowed us to map the 8443 port from vagrant to the host and match the certificate. When it changed to '10.0.2.15', that interface is not accessible from the host (vbox networking is a little strange, but there are basically two subnets, the guest->world (10.0.2.X) and guest -> host (192.168.42.X) ). When we try to connect to 10.0.2.15:8443, the host can't route to that network, and so kubectl would fail.

If this works, that's great, because it means minikube fixed the bug despite their previous lack of interest in fixing it. However, I don't want to find ourselves in the 'latest' trap again, so even if they have fixed it, we need to pin it to the version that's been fixed, not just rely on 'latest'.